common/js/src/logging/logging.js

```diff
@@ -49,6 +49,26 @@ goog.require('goog.log.Logger');
 goog.scope(function() {
 
 /** @const */
+var GSC = GoogleSmartCard;
```

Comment:
nit: use const directly (here and in other places).
Maybe it's worth a separate CL to change all instances in this file?

Reply:
Done - created #155 for this.